examples/bch-example/src/app/bch.root.example.ts

```diff
@@ -4,13 +4,13 @@ import { TatumBchSDK } from '@tatumio/bch'
 export async function exchangeRateExample() {
   const bchSDK = TatumBchSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
 
-  // Get currenct exchange rate
-  // You can find more details in https://apidoc.tatum.io/tag/Exchange-rate#operation/getExchangeRate
+  // Get the current exchange rate.
+  // https://apidoc.tatum.io/tag/Exchange-rate#operation/getExchangeRate
   const rate = await bchSDK.getExchangeRate()
```

Comment:
What's the difference between this example and the next one? By default, EUR is used for getting the exchange rate, so these two examples do the same thing?